lib/teaspoon/jasmine/integration.rb

```diff
@@ -0,0 +1,32 @@
+module Teaspoon
+  module Jasmine
+    JAVASCRIPT_VERSIONS = {
+      "1.3.1" => ["jasmine/1.3.0.js", "teaspoon-jasmine.js"],
+      "2.2.0" => ["jasmine/2.2.0.js", "teaspoon-jasmine.js"],
+    }
+
+    SUPPORT_LIBRARIES = [
+      "jasmine-jquery-1.7.0",
+      "jasmine-jquery-2.0.0",
```

Comment:
I think `jasmine-jquery-2.1.0` is the latest. That's what I've dropped in for j2.